charts/k3s/templates/statefulset.yaml

```diff
@@ -194,11 +197,24 @@ spec:
                 name: "vc-cidr-{{ .Release.Name }}"
                 key: cidr
           {{- end }}
+        ports:
+          - containerPort: 6443
+            protocol: TCP
+        {{- if .Values.vcluster.livenessProbe }}
+        livenessProbe:
+{{ toYaml .Values.vcluster.livenessProbe | indent 10 }}
+        {{- end }}
+        {{- if .Values.vcluster.readinessProbe }}
+        readinessProbe:
+{{ toYaml .Values.vcluster.readinessProbe | indent 10 }}
+        {{- end }}
         securityContext:
 {{ toYaml .Values.securityContext | indent 10 }}
         volumeMounts:
           - name: config
             mountPath: /etc/rancher
+          - name: kube
+            mountPath: /.kube
```

Comment:
The tests are failing because the `kube` volume is available only sometimes - `{{- if or .Values.securityContext.runAsUser .Values.securityContext.runAsNonRoot .Values.securityContext.readOnlyRootFilesystem }}` 
This needs to be fixed.

btw, would be nice to add the .kube volume to other charts too, but not a blocker.

Reply:
I don't think other charts need .kube volume, because they are not generating kubeconfig. I have tried k8s but it doesn't have this issue.